### DISCUSSION TITLE
A simplest workflow: One shortcut to trigger the dropbar then navi by hjkl

### DISCUSSION THREAD

**Discussion by LintaoAmons** [Bekaboo/dropbar.nvim#172] [Q&A]
Hi, I found it quite cumbersome when I want to use dropbar, I need to use shortcut to trigger `pick`, then take a look at the characters then found the one I want and press it.

I hope we can have a simplest workflow, which is a shortcut to trigger the dropbar at current buf's dir, then use `hjkl` to start navigate look around. (Most of the case, trigger the dropbar is because I want to look files within the same folder or folder nearby, so pick is too powerful and adds extra cognitive complexity)

I recorded how IDEA's breadcrumbs works, which is really simple~

![bread](https://github.com/user-attachments/assets/817c0aef-c571-4631-879e-2cb386e716d0)

**Comment by Bekaboo** [comment]
I think you will be interested in https://github.com/Bekaboo/dropbar.nvim/issues/160

**Reply by druskus20** [reply]
I am looking for something very similar. 
- h: menu back / component back
- l: menu forward
- C-h: **click previous component of the bar**, ignoring open menus (closing them)
- C-l: **click next component of the bar**, ignoring open menus (closing them)

I'm not quite sure how to access the previous/next "component" in bar. Any ideas?
